src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternPreferences.java

```diff
@@ -51,4 +56,9 @@ public boolean isEnforceLegalKey() {
     }
 
     public String getDefaultBibtexKeyPattern() { return defaultBibtexKeyPattern;}
+
+
```

Comment:
This one is probably my bad, although I know sometimes Eclipse add double empty lines.